frontend/src/components/footer.tsx

```
@@ -0,0 +1,53 @@
+               <span className="text-[#FF5B36]">K</span>aizen
+            </h1>
+          </Link>
+          <p>
+            Project: Kaizen
+            <br />
+            By ArcaneBytes @2025
+          </p>
+        </aside>
+        <nav>
+          <h6 className="footer-title">Services</h6>
+          <a className="link link-hover">Branding</a>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)